web/tests/libtest/env.py

```diff
@@ -257,15 +257,20 @@ def parts_to_url(codechecker_cfg):
 
 
 def get_workspace(test_id='test'):
-    tmp_dir = os.path.join(REPO_ROOT, 'build')
-    base_dir = os.path.join(tmp_dir, 'workspace')
-    if not os.path.exists(base_dir):
-        os.makedirs(base_dir)
+    """ return a temporary workspace for the tests """
+    workspace_root = os.environ.get("WORKSPACE")
```

Comment:
Jenkins uses the same environment variable. If the user runs tests in a jenkins job, this variable will always be set. I think it would be better to rename this environment variable to something like `CODECHECKER_TEST_WORKSPACE` or `CC_TEST_WORKSPACE`.

Please rename it in the Makefiles too.